Kernel/Heap/kmalloc.cpp

```diff
@@ -435,7 +436,12 @@ void kfree_sized(void* ptr, size_t size)
 
 size_t kmalloc_good_size(size_t size)
 {
-    return size;
+    VERIFY(size > 0);
+    for (auto const& slabheap : g_kmalloc_global->slabheaps) {
```

Comment:
Is it worth noting in a comment that the slab heaps and their sizes are constant, so we don't need to take any locks here?

Reply:
Sure, will add.